cloud-data-protection/src/components/register.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from "react";
+import './register.css';
+import {AuthService} from "../services/authService";
+import {Button, Input, Typography} from "@material-ui/core";
+import {useSnackbar} from 'notistack';
+import {http} from "../common/http";
+import snackbarOptions from "../common/snackbar/options";
+import { useHistory } from "react-router-dom";
+
+const Register = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [repeatPassword, setRepeatPassword] = useState('');
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const history = useHistory();
+
+    const authService = new AuthService();
+
+    const handleSubmit = async (e: any) => {
+        e.preventDefault();
+
+        const cancelToken = http.CancelToken.source().token;
+
+        const input = { email, password };
+
+        if (password !== repeatPassword) {
+            onError('Entered passwords do not match');
+            return;
+        }
+
+        await authService.register(input, cancelToken)
+            .then(() => onSuccess())
+            .catch((e: any) => onError(e))
+            .finally(() => onFinish());
+    }
+
+    const onSuccess = () => {
+        setEmail('');
+
+        enqueueSnackbar('Your account has been created. You can now log in using the specified credentials', snackbarOptions);
+
+        history.push('/login');
+    }
+
+    const onError = (e: any) => {
+        enqueueSnackbar(e, snackbarOptions);
+    }
+
+    const onFinish = () => {
+        setPassword('');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
        setPassword("");
```